contents/handbook/community/index.mdx

```diff
@@ -4,7 +4,28 @@ sidebar: Handbook
 showTitle: true
 ---
 
-Rather than using an off-the-shelf community platform, the [Website & Docs team](/handbook/small-teams/website-docs) has rolled our own. This gives us the flexibility to do what we want with it, all without having to depend on third parties or their cookies.
+We want to build a self-sustaining and scalable community of engaged users because it will enable us to own our audience in a way that third party social media platforms do not. Like brand or content, building a thriving community is a (very) long term bet, so we will need to both invest a lot of time up front and then wait to see what works and what doesn't. 
+
+Our approach to building community at PostHog differs from most devtools in two ways:
+
+- We are building our community around our _website and content_, rather than the product itself. This is because a) PostHog is a product that you add after you have already built something, and b) 90% of community activity ends up being support queries rather than general interaction. 
+- We are focusing on building the community _platform_ itself - creating the tools that enable the community to interact with each other, rather than hiring a community manager whose job it is to go out and talk to everyone on other platforms/social media - this is not scalable. 
+
+## Responsibility for community
+
+This is shared across multiple teams and people - we (deliberately) do not have one person responsible for 'community':
+
+- The [Website & Docs team](/handbook/small-teams/website-docs) builds the platform and tools. Rather than using an off-the-shelf community platform, we have rolled our own. This gives us the flexibility to do what we want with it, all without having to depend on third parties or their cookies.
+- The [Marketing team](/handbook/small-teams/marketing) doesn't 'run community' in the traditional sense, but is instead responsible for ensuring that the content hubs in particular have a steady stream of engaging content and replying to users when they engage. They also proactively respond to questions and use feedback to create new types of content such as tutorials and docs. 
```

Comment:
I feel like this is missing something. 

We say that 90% of community activity is support queries, but don't have any support element mentioned here or say which team takes responsibility for creating a self-serve community behaviour.

There's a if-you-build-it-they-will-come argument that by merely creating the tools W&D is facilitating this, but I think this is only going to take us so far and that Support (basically Marcus) need to be involved. 

This is because the actual best experience users can have is to not contact us through the community at all, but through the in-app support. This is what all of our engineering teams prefer, and is what gets users a guaranteed response. It's also what we most regularly direct users to. 

So, if we want to grow the community and redirect some users out of the in-app channel (_do_ we want to do that?) then Support responses are the best place to do it. 

I also think that if we're outlining a vision then we need to clarify what types of engagements we want to facilitate within the community. We're saying 90% of activity is support queries, but the main way we engage is by writing non-support content. Is our goal to just cater to the 10%, or do we want the community to stop being a place people ask for support? What sort of engagements do we want to encourage?

Reply:
I'll have another go at rephrasing it - my intention here was that support is a totally separate thing that is emphatically _not_ part of what we consider community, but it sounds like I didn't get that across successfully!